assets/dev/js/editor/controls/base.js

```diff
@@ -126,6 +126,13 @@ ControlBaseView = Marionette.CompositeView.extend( {
 
 		var isVisible = elementor.helpers.isActiveControl( this.model, settings.attributes, settings.controls );
 
+		const parentContainerType = this.model.get( 'parent_container_type' );
+
+		if ( isVisible && parentContainerType ) {
+			isVisible = this.container.settings.get( 'container_type' ) === parentContainerType ||
```

Comment:
I am not sure about it but should the `ControlBaseView` be aware of the `this.container`?